src/components/ContentWidgets/Accordion/AccordionEditor.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { WidgetEditorProps } from '../../ContentMapping/ContentMapping';
+import { Accordion } from './Accordion';
+
+export const AccordionEditor: React.FC<WidgetEditorProps> = ({
+    editedContent,
+    setEditedContentOnChange
+}) => {
+    const emptySection = {
+        title: "",
+        panel: ""
+    }
+    const accordionContent = editedContent.accordion_content || [emptySection];
+    
+    const addAccordionSection = (index: number) => {
```

Comment:
For testing you might want to have these functions outside the component function instead